qiskit_nature/problems/second_quantization/lattice/lattice/lattice.py

```diff
@@ -0,0 +1,256 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""General Lattice."""
+from dataclasses import asdict, dataclass
+from typing import Callable, List, Optional, Sequence, Tuple, Union
+
+import numpy as np
+from qiskit.exceptions import MissingOptionalLibraryError
+from qiskit.tools.visualization import HAS_MATPLOTLIB
+from retworkx import NodeIndices, PyGraph, WeightedEdgeList, adjacency_matrix
+from retworkx.visualization import mpl_draw
+
+if HAS_MATPLOTLIB:
+    # pylint: disable=unused-import
+    from matplotlib.axes import Axes
+    from matplotlib.colors import Colormap
+
+
+@dataclass
+class DrawStyle:
```

Comment:
Maybe we should change this name to indicate that this is a utility class for _lattices_. E.g.` LatticeDrawStyle`.
That would also make it more obvious why this class is part of the `lattice.py` file

Reply:
I agree with the name change, so I have changed it to `LatticeDrawStyle`.